.github/ISSUE_TEMPLATE/2-bug-android.yml

```diff
@@ -0,0 +1,115 @@
+        please contact support instead of filing a bug report. Go to `Settings -> Support -> Report a problem`.
+        That way the support team gets redacted logs from your app and can help you out better.
+        You can also just email them at support@mullvad.net.
+      options:
+        - label: I know this is an issue with the app, and contacting Mullvad support is not relevant.
+          required: true
+
+  - type: checkboxes
+    id: checked-other-issues
+    attributes:
+      label: I have checked if others have reported this already
+      description: >
+        Before you submit a bug report, please look through the
+        [existing issues](https://github.com/mullvad/mullvadvpn-app/issues?q=is%3Aissue)
+        to see if it has already been reported by others. If so, please comment in those threads instead
+        of creating new ones.
+      options:
+        - label: I have checked the issue tracker to see if others have reported similar issues.
+          required: true
+
+  - type: textarea
+    id: current-behavior
+    attributes:
+      label: Current Behavior
+      description: What is the current behavior you experience that you think is not correct?
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected Behavior
+      description: What is the behavior that you expect to happen instead?
+    validations:
+      required: true
+
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Steps to Reproduce
+      description: >
+        Please provide clear and detailed steps on how to reproduce the issue you are reporting.
+        If it is very hard to reproduce the issue, then there is no guarantee we can locate the bug and fix it.
+      value: |
+        1. ...
+        2. ...
+    validations:
+      required: true
+
+  - type: textarea
+    id: logs
+    attributes:
+      label: Failure Logs
+      description: >
+        If relevant, please include logs from the app from the time around when the bug manifested itself.
+
+        Go to settings (cogwheel) -> Support -> Report a problem -> View app logs to see the logs and
+        copy them to here.
+      render: shell
+
+  - type: input
+    id: os-version
+    attributes:
+      label: Android version
+      description: >
+        On what version(s) of Android have you experienced this bug?
+        If you have experienced it on multiple versions you can write more than one version here.
+
+        Please also include on what Android ROM you have seen this.
+        For example if you run stock Android from your phone vendor, Graphene, LineageOS or similar.
+
+  - type: input
+    id: device-model
+    attributes:
+      label: Device model
+      description: >
+        On what device have you seen this bug? for example "Samsung S22" or "Pixel 7"
+        If you have experienced it on multiple versions you can write more than one version here.
```

Comment:
"model" instead of "version" to keep things consistent? (with the label and the previous sentence)
Beside there is already the "Android version" section just before.